DESCRIPTION

```diff
@@ -14,6 +14,6 @@ Depends: R (>= 3.3)
 License: MIT + file LICENSE
 LazyData: true
 Suggests: testthat, tinytest
-Imports: jsonlite, httr, R6, base64enc
+Imports: jsonlite, arrow, httr, R6, base64enc
```

Comment:
Just FYI for the CI depends:  arrow is both somewhat expensive to build, and somewhat fragile.  Maybe keep an eye on this one.

Reply:
Awesome context -- thanks @eddelbuettel !! :D